coverage/base.css

```diff
@@ -0,0 +1,212 @@
+body, html {
```

Comment:
I don't think you need the coverage folder on github, you should consider adding it to `.gitignore`